### ISSUE TITLE
Protect static pages

### ISSUE THREAD

**Issue by ahopkins** [ahopkins/sanic-jwt#40]
Related to #38. Perhaps add examples on how to protect static pages.

**Comment by amor71** [comment]
@ahopkins I've read through the initial issue (and you suggested a solution), as a 'first timer' can I try to tackle this example, however, my 2-cents is that this functionality should be handled by a reverse-proxy to check for the presence of authentication and handle re-directs. Let me know if you think otherwise, and how could I help

**Comment by ahopkins** [comment]
@amor71 I do not disagree. If I were to handle this personally I would not in this manner. But, that is because it is my opinion that Sanic is best for handling Web APIs. I would not personally use it (for example) in conjunction with jinja to serve up HTML content. This doesn't mean that you couldn't, just that I would not.

Now I honestly have never even tried. I guess maybe an option on the `protected` decorator` that looks  at `LOGIN_REDIRECT_URL` from config. 

Something that would look like this:

```python
@protected(redirect_on_fail=True)
async def protected_html(request):
    return html(...)
```

Or, with an option to override `LOGIN_REDIRECT_URL`:

```python
@protected(redirect_on_fail=True, redirect_url='go_here.html')
async def protected_html(request):
    return html(...)
```

**Comment by amor71** [comment]
cool! I will look into it

**Comment by ahopkins** [comment]
#153 is merged, and this will be in v1.3
